src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,132 @@
+@import "./blocks/variables";
+
+.watch {
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: 3px dashed black;
+  border-radius: 50%;
+  margin: 50px auto;
+  background: radial-gradient(circle, rgb(253, 253, 65), rgb(226, 17, 17));
+
+  & :nth-child(n) {
+    position: absolute;
+  }
+
+  &__minutes {
+    top: $watch-size / 2 - $minutes-height;
+    left: ($watch-size - $minutes-width) / 2;
+    width: $minutes-width;
+    height: $minutes-height;
+    border-radius: 5px;
+    background-color: red;
+  
+    transform-origin: 7.5px 100px;
+    animation-name: rotateMinute;
```

Comment:
You can use animation shorthand(instead of 4 row of code(25-28)), or move this to mixin. Also please change this for second-hand animation. 
https://developer.mozilla.org/en-US/docs/Web/CSS/animation